LICENSE

```diff
@@ -0,0 +1,11 @@
+Copyright (c) 2021 niriPipe Developers All rights reserved.
```

Comment:
What licence is this?

Reply:
This is my attempt to copy the 3-clause [BSD license in astroquery](https://github.com/astropy/astroquery/blob/master/LICENSE.rst). However, I'm far from certain which license I should use for an open source CADC project (and if copyright should be assigned to something like `niriPipe developers`, or something like `CADC`. 
Do you think a simple MIT license would be better, with copyright either assigned to `niriPipe developers`, myself, or `CADC`?

Reply:
Pick the `LICENCE` file from one of the existing `opencadc` projects

Reply:
Great, thank you. Done in 7e546ad0.